src/ensembl/src/content/app/species/components/species-sidebar/SpeciesSidebar.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
+
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getActiveGenomeSidebarPayload } from 'src/content/app/species/state/sidebar/speciesSidebarSelectors';
+
+import { RootState } from 'src/store';
+import {
+  SpeciesSidebarPayload,
+  fetchSidebarPayload
+} from 'src/content/app/species/state/sidebar/speciesSidebarSlice';
+
+import styles from './SpeciesSidebar.scss';
+
+type Props = {
+  activeGenomeId: string | null;
+  sidebarPayload?: SpeciesSidebarPayload | null;
+  fetchSidebarPayload: () => void;
+};
+
+const SpeciesSidebar = (props: Props) => {
+  useEffect(() => {
+    if (!props.sidebarPayload) {
+      props.fetchSidebarPayload();
+    }
+  }, [props.sidebarPayload, props.activeGenomeId]);
+
+  if (!props.sidebarPayload) {
+    return <div>No data to display</div>;
+  }
+  const payload = props.sidebarPayload;
+
+  type AnnotationEntry = {
+    label: string;
+    value?: string | null;
+    url?: string;
+  };
+  const assemblyDate = new Date(payload.assembly_date);
+  const formattedAssemblyDate = Intl.DateTimeFormat('en-GB', {
+    month: 'short',
+    year: 'numeric'
+  }).format(assemblyDate);
+  const annotationEntriesMap: AnnotationEntry[][] = [
```

Comment:
It's not even a map 😕 . `annotationEntriesMap` -> `annotationEntries`?

Reply:
That's done